Assets/HoloToolkit/Utilities/Scripts/DebugPanelControllerInfo.cs

```diff
@@ -28,6 +28,34 @@ private class ControllerState
             public Vector2 TouchpadPosition;
         }
 
+        // Text display label game objects
+        public TextMesh leftInfoTextPointerPosition;
```

Comment:
I didn't catch this last time, but please capitalize the first letter of all the public variables here.